docs/topics/logging.txt

```diff
@@ -219,9 +219,17 @@ configuration merged with Django's defaults, hence you can decide if you want to
 add to, or replace the existing configuration.
 
 If the ``disable_existing_loggers`` key in the :setting:`LOGGING` dictConfig is
-set to ``True`` (which is the default) the default configuration is completely
-overridden. Alternatively you can redefine some or all of the loggers by
-setting ``disable_existing_loggers`` to ``False``.
+set to ``True`` (which is the default) then all loggers from the default
+configuration will be disabled. Alternatively you can redefine some or all of
```

Comment:
"...will be disabled. This is _not_ the same as removed; the logger will still exist, but will silently discard anything logged to it, not even propagating it to a parent logger. Thus you should be very careful using `disable_existing_loggers: True`; it's probably not what you want. Instead, you can set `disable_existing_loggers` to `False` and redefine some or all of the default loggers; or you can set `LOGGING_CONFIG` to `None` and handle logging config yourself."